src/server.c

```diff
@@ -2519,111 +2519,16 @@ void commandAddSubcommand(struct redisCommand *parent, struct redisCommand *subc
     serverAssert(dictAdd(parent->subcommands_dict, sdsnew(subcommand->name), subcommand) == DICT_OK);
 }
 
-
-/* Parse the flags string description 'strflags' and set them to the
- * command 'c'. Abort on error. */
-void parseCommandFlags(struct redisCommand *c, const char *strflags) {
-    int argc;
-    sds *argv;
-
-    /* Split the line into arguments for processing. */
-    argv = sdssplitargs(strflags,&argc);
-    if (argv == NULL)
-        serverPanic("Failed splitting strflags!");
-
-    for (int j = 0; j < argc; j++) {
-        char *flag = argv[j];
-        if (!strcasecmp(flag,"write")) {
-            c->flags |= CMD_WRITE|CMD_CATEGORY_WRITE;
-        } else if (!strcasecmp(flag,"readonly")) {
-            c->flags |= CMD_READONLY|CMD_CATEGORY_READ;
-        } else if (!strcasecmp(flag,"denyoom")) {
-            c->flags |= CMD_DENYOOM;
-        } else if (!strcasecmp(flag,"admin")) {
-            c->flags |= CMD_ADMIN|CMD_CATEGORY_ADMIN|CMD_CATEGORY_DANGEROUS;
-        } else if (!strcasecmp(flag,"pubsub")) {
-            c->flags |= CMD_PUBSUB|CMD_CATEGORY_PUBSUB;
-        } else if (!strcasecmp(flag,"noscript")) {
-            c->flags |= CMD_NOSCRIPT;
-        } else if (!strcasecmp(flag,"random")) {
-            c->flags |= CMD_RANDOM;
-        } else if (!strcasecmp(flag,"sort_for_script")) {
-            c->flags |= CMD_SORT_FOR_SCRIPT;
-        } else if (!strcasecmp(flag,"loading")) {
-            c->flags |= CMD_LOADING;
-        } else if (!strcasecmp(flag,"stale")) {
-            c->flags |= CMD_STALE;
-        } else if (!strcasecmp(flag,"skip_monitor")) {
-            c->flags |= CMD_SKIP_MONITOR;
-        } else if (!strcasecmp(flag,"skip_slowlog")) {
-            c->flags |= CMD_SKIP_SLOWLOG;
-        } else if (!strcasecmp(flag,"asking")) {
-            c->flags |= CMD_ASKING;
-        } else if (!strcasecmp(flag,"fast")) {
-            c->flags |= CMD_FAST | CMD_CATEGORY_FAST;
-        } else if (!strcasecmp(flag,"no_auth")) {
-            c->flags |= CMD_NO_AUTH;
-        } else if (!strcasecmp(flag,"may_replicate")) {
-            c->flags |= CMD_MAY_REPLICATE;
-        } else if (!strcasecmp(flag,"sentinel")) {
-            c->flags |= CMD_SENTINEL;
-        } else if (!strcasecmp(flag,"only_sentinel")) {
-            c->flags |= CMD_SENTINEL; /* Obviously it's s sentinel command */
-            c->flags |= CMD_ONLY_SENTINEL;
```

Comment:
same

Reply:
this is no longer required

Reply:
how come? how do you distinguish between ones that are "also in sentinel" and ones that are "only in sentinel"?

Reply:
ohh, i see you just listed both in the json file.
isn't it cleaner to just list the SENTINEL_ONLY in the json file, and have the other one implicit?

Reply:
i think i prefer the only implicit thing here is ACL categories and not command flags... we can revisit that in the future

specifically for SENTINEL I doubt there would be other "only_sentinel" commands

ok?